### DISCUSSION TITLE
Downloaded ThreeFingerDragOnWindows but don't know how to install

### DISCUSSION THREAD

**Discussion by shula16** [community/community#127065] [New to GitHub] [answered]
Hi! I would really appreciate someone helping me on this question.
I downloaded the ThreeFingerDragOnWindows here: https://github.com/ClementGre/ThreeFingerDragOnWindows
Pressed the green button that writes "code" and then pressed "download zip."
Now I got the file downloaded but upon opening it, I don't know which file in it to click and install.
I've tried basically all of them but didn't find one that works.
<img width="913" alt="Screenshot 2024-06-03 at 6 24 04 PM" src="https://github.com/community/community/assets/171545364/da6fb380-2a45-4d0d-b4e3-388d4b8449d8">
Please kindly help me. Thanks!

P.s. the screenshot is my mac's screen because I don't have my windows with me right now.

**Comment by ExpiredWater** [comment, marked as answer]
Hi @shula16,
It looks like you have download the source code that need to be compiled. You can download pre compiled binaries from the Releases tab on the repository page. This is a direct link for the page in question https://github.com/ClementGre/ThreeFingerDragOnWindows/releases

Also the application can be installed from the MS store which comes with automatic updates, using this link https://apps.microsoft.com/detail/threefingersdragonwindows/9MSX91WQCM2V

I hope this solves your issue.

**Reply by shula16** [reply]
Very grateful for your help. Yes the MS store one perfectly solves my issue and is even much better than what I asked for!  
Thank you for your kind help!

**Reply by ethanxo** [reply]
To add to this, developers will often post compiled executables or installers, if applicable, in the Releases section under the repository description / stats. If there is no .exe/.msix/.msi file in the releases section assume you must build it yourself (simple as opening the project and pressing build 95% of the time.)
